docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -25,7 +25,7 @@ Feel free to submit a pull request!
 
 ### Layout
 
-- [@mui-treasury/layout](https://mui-treasury.com/layout/): Components to handle the overall layout of a page. Check out examples such as [a reactjs.org clone](https://mui-treasury.com/layout/clones/reactjs/).
+- [@mui-treasury/layout](https://mui-treasury.com/?path=/docs/layout-introduction--docs): Components to handle the overall layout of a page. Check out examples such as [a reactjs.org clone](https://mui-treasury.com/?path=/story/layout-app-reactlegacy--react-legacy).
```

Comment:
Nice 👍 

I pushed this even further in 01095d89eaff366c55ac3e238543c539cfb701f1.